config/routes.rb

```diff
@@ -1320,17 +1320,19 @@
       )
     },
 
-    :physical_storage    => {
-      :get  => %w(
+    :physical_storage   => {
+      :get  => %w[
         download_data
-        show_list
```

Comment:
Are you sure this doesn't break anything?

Reply:
seems that the problem is the representation of the github.
all routes of the physical storage:
```ruby
:physical_storage   => {
      :get  => %w[
        download_data
        download_summary_pdf
        show
        show_list
        new
      ],
      :post => %w[
        listnav_search_selected
        quick_search
        show_list
      ] + adv_search_post + save_post + exp_post
    },
```

You can see it in the full source code